devices/Bmm150/README.md

```diff
@@ -0,0 +1,181 @@
+# Bmm150 - Magnetometer
```

Comment:
Can you please have a look at the readmes in [.NET IoT](https://github.com/dotnet/iot) and use the same flow. We've been reworking them and not yet implemented this flow on nanoFramework. You have most of the content, it's just about reorganizing the file a little bit

Reply:
I renamed the 2nd header to Documentation (same as .net iot readmes). Although, we do have more content in this readme in order to provide more details about the tests.